pinc/ProjectTransition.inc

```diff
@@ -1168,11 +1184,78 @@ new ProjectTransition(
         'action_name'           => _("Return to Post-Processor"),
         'confirmation_question' => _("Are you sure you want to return this book to the post-processor for further work?"),
         'detour'                => null,
-        'settings_template'     => "modifieddate='<TIMESTAMP>', checkedoutby=postproofer, postproofer=''",
-        'collateral_actions'    => null,
+        // formerly ppverifier was only set when the project was uploaded to PG
+        // it is set here so the PPer can return it to the same PPVer
+        // any transition out of PROJ_POST_FIRST_CHECKED_OUT sets it to ''
+        // this will not affect how the PPVer is ultimately set.
+        'settings_template'     => "modifieddate='<TIMESTAMP>', checkedoutby=postproofer, postproofer='', ppverifier=checkedoutby",
+        'collateral_actions'    => "return_to_PP_collateral",
     )
 );
 
+function return_to_PP_collateral( $old_project, $project, $transition, $who )
+{
+    global $site_abbreviation;
+
+    $PPer_username = $project->checkedoutby;
+
+    $PPer = new User($PPer_username);
+    $PPer_email_addr = $PPer->email;
+    $body_blurb =
+        $project->email_introduction() .
+        "\nThis project has been returned to you for further work.\n";
```

Comment:
This and the subject need to be localized in the receiver's language. See how `round_to_PP_avail_collateral()` does it. `return_from_PP_to_PPV_collateral()` need translation too.

Reply:
Done these.